samples/getting_started/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import {v2 as cloudinary} from 'cloudinary';
+
+// Update to your credentials
+cloudinary.config({
+    cloud_name: '<cloud_name>',
+    api_key: '<api_key>',
+    api_secret: '<api_secret>'
+});
+
+// Ensure that you're registered to auto tagging https://cloudinary.com/documentation/google_auto_tagging_addon
+// Ensure that you're registered to background removal https://cloudinary.com/documentation/cloudinary_ai_background_removal_addon
+//Upload an image with auto tag
+//apply eager transformation
+cloudinary.uploader.upload(
+    "https://res.cloudinary.com/demo/image/upload/v1707306308/cld-docs-hp/walking_woman",
+    {
+        public_id: "walking_woman",
+        categorization: "google_tagging",
+        eager: [{effect: "background_removal"}]
```

Comment:
Not sure there's any reason to do the background_removal as eager, since it's supported as an on-the-fly transformation?  I think that can be confusing for a new user that we have both when it's not really essential?